tests/test001.sh

```diff
@@ -0,0 +1,104 @@
+#! /bin/bash
+#
+# Minimal test established a peer link between two meshd instances over hwsim
+#
+
+[ $(uname) = "Linux" ] || { echo "This test only runs on Linux"; exit 1; }
+
+cleanup() {
+    sudo killall meshd-nl80211
+    rm -fr "${TMP1}" "${TMP2}"
+}
+
+trap cleanup SIGINT
+
+sudo modprobe -r mac80211-hwsim &> /dev/null
+sudo modprobe mac80211-hwsim radios=4 || { echo "Failed to load mac80211-hwsim module."; echo 1; }
+
+sudo iw phy phy2 interface add mesh0 type mesh
```

Comment:
Every time this comes up I think, "someone should write a shell function to dig around in sysfs and find which phys belong to hwsim" :)

Reply:
To be clear, no one has as far as I know, but I could have used it before... tests always break when you also have hardware nics...